src/server.js

```diff
@@ -13,6 +13,10 @@ import loader from 'taskcluster-lib-loader'
 import taskcluster from 'taskcluster-client'
 import flash from 'connect-flash'
 import scanner from './scanner'
+import v1 from './v1'
+import tc_lib_app from 'taskcluster-lib-app'
```

Comment:
I'm under the impression we should prefer camelCase in our codebase, but I'm definitely not an arbiter of taste here.